### DISCUSSION TITLE
Argo Workflow - share machine between steps

### DISCUSSION THREAD

**Discussion by rajeshvkothari3003** [argoproj/argo-workflows#9106] [Q&A] [answered]
I am using containerSet feature of argo-workflows. However, due to some reason I want to run steps in the same docker container instead of using multiple docker containers in pod. Also need to display steps that run on same container in Argo GUI.
How can I implement it?

Thanks!

**Comment by ABBARNABIL** [comment]
You can add a workflow template into argo using the GUI , and do a submit on it using a post request.

**Reply by rajeshvkothari3003** [reply]
Yes, I am able to do that. But what I trying to do is something different. I have updated my question.

**Comment by terrytangyuan** [comment, marked as answer]
Argo Workflows only provides container/pod level steps.
